source/adminguide/ui.rst

```diff
@@ -273,6 +280,47 @@ Some assorted primary theme colours:
 - Green: #52C41A
 - Purple: #722ED1
 
+Contextual help documentation URLs can be customized with the help of `docBase` and `docHelpMappings` properties.
+A list of documentation help suffixes in code can be obtained using following command:
+
+.. parsed-literal::
+
+   grep -rnw cloudstack/ui/src/config/section -e 'docHelp:'
+
+Below example shows configuration changes for custom documentation help URLs:
+
+.. parsed-literal::
+
+   {
+      ...
+      "docBase": http://mycustomwebsite.com,
+      ...
+      "docHelpMappings": {
+         "adminguide/virtual_machines.html": "custom_vm_page.html",
+         "adminguide/templates.html": "custom_templates_page.html"
+      },
+      ...
+   }
+
+UI also provides option to show custom plugins by displaying a custom HTML page or service in an iframe. Such plugins can be listed in the config file using `plugins` property.
+Example for adding a custom plugin:
+
+.. parsed-literal::
+
+   {
+      ...
+      plugins: [
+         {
+            "name": "ExamplePlugin",
+            "icon": "appstore",
+            "path": "example.html"
```

Comment:
@shwstppr can you add another example, where the path is an external URL; for example:
```
  "plugins": [{
    "name": "External Link Example",
    "icon": "hdd",
    "path": "http://192.168.1.10:7000"
  }]

```

Reply:
@rhtyd done